README.md

```diff
@@ -1,66 +1,37 @@
+This repository contains a shared configurations for use across all GEWIS projects. It aims to enforce consistent coding standards and code quality.
 
-## Installation
-
-To use this ESLint configuration in your project, install it using `npm` or `yarn`.
-```bash
-npm install @gewis/eslint-config@github.com:GEWIS/eslint-config --save-dev
-yarn add @gewis/eslint-config@github.com:GEWIS/eslint-config --dev
-```
-
-## Usage
-The package provides sharable configurations for [ESLint](https://eslint.org/docs/latest/extend/shareable-configs) and
-[prettier](https://prettier.io/docs/en/sharing-configurations) that can be used directly in your project's configuration files. 
+- [ESLint](https://eslint.org/docs/latest/extend/shareable-configs) configuration for JavaScript, TypeScript, React and Vue
+- [Prettier](https://prettier.io/docs/en/sharing-configurations) configuration including ESLint ignores
 
 > [!NOTE]  
 > The shared configurations for ESLint are not compatible with ESLint `v8.x`
 
-### ESLint Configuration
-
-In your project's root directory, create or update the `eslint.config.{js,mjs,cjs,ts,mts,cts}` file to extend this configuration. 
+### ESLint 
+In your project's root directory, create or update the `eslint.config.{js,mjs,cjs,ts,mts,cts}` file to extend this configuration.
 You can pick a combination of various configuration files:
 
-- `eslint.common.mjs`
+- `index.mjs`
 - `eslint.vue.mjs`
-- `eslint.prettier.mjs`
+- `index.mjs`
 - `eslint.react.mjs`
 
-```javascript
-import eslint from '@gewis/eslint-config/eslint.common.mjs';
-import vue from '@gewis/eslint-config/eslint.vue.mjs';
-import prettier from '@gewis/eslint-config/eslint.prettier.mjs';
-
-export default [...eslint, ...vue, ...prettier];
-```
-
-It is possible to (partially) [override settings](https://eslint.org/docs/latest/extend/shareable-configs#overriding-settings-from-shareable-configs)
+Then, follow one of the specific README's to configure your ESLint instance. It is possible to (partially) [override settings](https://eslint.org/docs/latest/extend/shareable-configs#overriding-settings-from-shareable-configs)
 of the shared configurations in your config file. It is also possible to add your own ignored file to the configurations
 
 > [!NOTE]  
 > All files in the `.gitignore` file(s) are ignored by default
 
-### Prettier Configuration
-
-In your project's root directory, create or update the `.prettierrc.mjs` file to extend this configuration. 
-There is a single [shared prettier configuration](https://prettier.io/docs/en/sharing-configurations) available; `prettier.mjs`.
-
-```javascript
-import prettier from '@gewis/eslint-config/prettier.mjs';
+### Prettier
 
-/**
- * @type {import("prettier").Config}
- */
-export default {
-  ...prettier,
-};
-```
+In your project's root directory, create or update the `.prettierrc.mjs` file to extend this configuration.
```

Comment:
minimal example? or link to the readme

Reply:
No thank you, then you will have to keep those references up to date down the line.